src/mumble/ACLEditor.cpp

```diff
@@ -115,6 +115,7 @@ ACLEditor::ACLEditor(int channelid, const MumbleProto::ACL &mea, QWidget *p) : Q
 		QString name = ChanACL::permName(perm);
 
 		if (! name.isEmpty()) {
+			if ( (g.sh->uiVersion < 0x010400) && ((perm == 0x100000) || (perm == 0x200000)) ) continue;
```

Comment:
Please add a comment about why these permissions will be skipped.

Also I think we shouldn't use the numeric values but the actual named enum entries here. That'll increase readability :point_up: